python/kvikio/kvikio/benchmarks/utils.py

```diff
@@ -45,15 +45,15 @@ def pprint_sys_info() -> None:
             dev = pynvml.nvmlDeviceGetHandleByIndex(0)
 
         if dev is not None:
+            gpu_name = f"{pynvml.nvmlDeviceGetName(dev)} (dev #0)"
             try:
-                pynvml.nvmlDeviceGetMigMode(dev)
-            except pynvml.NVMLError:
-                with contextlib.suppress(pynvml.NVMLError):
-                    gpu_name = f"{pynvml.nvmlDeviceGetName(dev)} (dev #0)"
-                    mem_total = format_bytes(pynvml.nvmlDeviceGetMemoryInfo(dev).total)
-                    bar1_total = format_bytes(
-                        pynvml.nvmlDeviceGetBAR1MemoryInfo(dev).bar1Total
-                    )
+                mem_total = format_bytes(pynvml.nvmlDeviceGetMemoryInfo(dev).total)
+            except pynvml.NVMLError_NotSupported:
+                mem_total = "Device has no memory resource"
+            try:
+                bar1_total = pynvml.nvmlDeviceGetBAR1MemoryInfo(handle).bar1Total
```

Comment:
This gets flagged by the style check [on CI]( https://github.com/rapidsai/kvikio/actions/runs/15196629457/job/42742221569?pr=729#step:7:227 )

```
python/kvikio/kvikio/benchmarks/utils.py:54:65: F821 undefined name 'handle'
```

Guessing we want to use `dev` here as used in the rest of the code

```suggestion
                bar1_total = pynvml.nvmlDeviceGetBAR1MemoryInfo(dev).bar1Total
```

Reply:
Good catch, thanks @jakirkham !